components/list_item/list_item.vue

```diff
@@ -224,14 +243,15 @@ export default {
   border-radius: var(--size-300);
 }
 
-.dt-list-item--focusable:focus,
-.dt-list-item--focusable:focus-within,
 .dt-list-item--highlighted {
   background-color: hsla(var(--black-900-h), var(--black-900-s), var(--black-900-l), 0.05);
 }
 
-.dt-list-item--highlighted:active {
-  background-color: hsla(var(--black-900-h), var(--black-900-s), var(--black-900-l), 0.1);
+.dt-list-item--selected {
+  background-color: var(--purple-100);
```

Comment:
Remove the background color, the presence of the check icon is sufficient. Plus, there may be multiple checked items and a background color muddies it. For example...

<img width="353" alt="image" src="https://user-images.githubusercontent.com/1165933/217564652-e6a1674c-ed5f-40e1-949d-bb86897c836a.png">